src/components/templates/SimpleTable/SimpleTableManual.tsx

```diff
@@ -369,10 +370,10 @@ export default function SimpleTableManual(props) {
           </TableBody>
         </Table>
         {readOnlyMode && rowData && rowData.length === 0 && (
-          <div className='no-records'>No records found.</div>
+          <div className='no-records' id='no-records'>No records found.</div>
```

Comment:
Although I approved the PR but here, both id's are identical which should be avoided.
Probably you can use, something like this  'readonly-no-records', 'editable-no-records'.

Reply:
yes id's should n't be identical but both div's will not render at once.